### PR TITLE
Improve products documentation

### PR DESCRIPTION
**Pull request by gonzunigad**
Improve products documentation

### CODE REVIEW

File: producto/webpay/README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Un flujo de pago en Webpay generalmente cuenta con los siguientes pasos:
 
 1. El tarjetahabiente selecciona los productos o servicios a pagar en el sitio del comercio.
 
-2. El tarjetahabiente elige pagar con Webpay en donde, dependiendo de los productos contratados por el comercio, se despliegan las alternativas de pago de crédito con productos cuotas y/o débito Redcompra.
+2. El tarjetahabiente elige pagar con Webpay en donde, dependiendo de los productos contratados por el comercio, se despliegan las alternativas de pago: Tarjeta de crédito, débito Redcompra y/o prepago.
 
-3. Durante el proceso de pago el emisor de la tarjeta autentica al tarjetahabiente antes de realizar la transacción financiera, con el objetivo de validar que la tarjeta este siendo utilizada por el titular.
+3. Durante el proceso de pago el emisor de la tarjeta autentica al tarjetahabiente antes de realizar la transacción financiera, con el objetivo de validar que la tarjeta esté siendo utilizada por el titular.
```
Comment (isabellaahc) [inline producto/webpay/README.md:15]:
```suggestion
3. Durante el proceso de pago, el emisor de la tarjeta autentica al tarjetahabiente antes de realizar la transacción financiera, con el objetivo de validar que la tarjeta esté siendo utilizada por el titular.
```

File: producto/webpay/README.md [hunk 2]
```diff
@@ -28,31 +28,39 @@ A continuación se describen los productos que se ofrecen en los servicios web d
...
+El concepto de "mall" agrupa múltiples tiendas. Son dichas tiendas las que pueden generar transacciones. Tanto el mall 
+como cada una de las tiendas asociadas son identificadas por un código de comercio diferente.
 
-El concepto de "mall" agrupa múltiples tiendas. Son dichas tiendas las que pueden generar transacciones. Tanto el mall como cada una de las tiendas asociadas son identificadas por un código de comercio diferente.
-
-El tipo de transacción Mall es útil para proveedores de servicios tecnológicos (PST) que pueden realizar una única integración con Transbank y realizar cobros a nombre de los clientes de dichos servicios tecnológicos. Por ejemplo una plataforma SaaS puede ser un mall y las empresas clientes de la plataforma pueden ser las tiendas de dicho mall. De esa manera, la recaudación que la plataforma realice irá directo al cliente a nombre del cual realizó cada cobro.
+El tipo de transacción Mall es útil para proveedores de servicios tecnológicos (PST) que pueden realizar una única 
+integración con Transbank y realizar cobros a nombre de los clientes de dichos servicios tecnológicos. Por ejemplo una
```
Comment (isabellaahc) [inline producto/webpay/README.md:47]:
```suggestion
integración con Transbank y realizar cobros a nombre de los clientes de dichos servicios tecnológicos. Por ejemplo, una 
```

**Review by isabellaahc — COMMENTED** [review]
Please check "," suggestions

**Comment by gonzunigad** [comment]
@isabellaahc changes made. Can you review it again? please :)

**Review by ffiebig — APPROVED** [review]

**Review by isabellaahc — APPROVED** [review]
LGTM

**Pull request merged**